viam-cartographer_internal_test.go

```diff
@@ -428,6 +430,83 @@ func TestGetPointCloudMapEndpoint(t *testing.T) {
 	})
 }
 
+func setMockGetPointCloudFunc(
+	mock *cartofacade.Mock,
+	pc []byte,
+) {
+	mock.GetPointCloudMapFunc = func(
+		ctx context.Context,
+		timeout time.Duration,
+	) ([]byte, error) {
+		return pc, nil
+	}
+}
+
+func TestGetPointCloudMapEndpointModularizationV2Endpoint(t *testing.T) {
+	svc := &cartographerService{Named: resource.NewName(slam.API, "test").AsNamed()}
+	mockCartoFacade := &cartofacade.Mock{}
+
+	svc.cartofacade = mockCartoFacade
+	svc.modularizationV2Enabled = true
+
+	inputPointCloudMapBytes := []byte{}
```

Comment:
I think you can delete this variable. Why do you need to share this empty byte slice between tests?

In general I try to have all test setup variables which are used by tests always be immutable, to prevent one test from affecting another test.

Reply:
leftover from trying to match old tests - removed!